src/main/lrsql/system.clj

```diff
@@ -2,6 +2,7 @@
   (:require [com.stuartsierra.component :as component]
             [lrsql.system.database :as db]
```

Comment:
Now that we have a separate DB-centric config in tuning, having it outside the `system.database` namespace can get confusing. Either move the tuning record to `system.database` or rename it to `system.connection`, since that is the record that namespace is currently focused on.

Reply:
see above.